src/lib/consensus/proof_of_stake.ml

```diff
@@ -1551,7 +1558,7 @@ module Make_str (A : Wire_types.Concrete) = struct
               (Global_slot.t * Global_slot.t list) Quickcheck.Generator.t =
             let open Quickcheck.Generator in
             let open Quickcheck.Generator.Let_syntax in
-            let module GS = Mina_numbers.Global_slot in
+            let module GS = Mina_numbers.Global_slot_since_genesis in
```

Comment:
I think this should also be `Global_slot_since_hard_fork.t`? if it is used for since_genesis as well, then you may have to add one for since_genesis